front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChartV2.tsx

```diff
@@ -1,343 +0,0 @@
-import { createTrainV2 } from 'modules/simulationResult/components/SpaceTimeChart/createTrain';
-import { drawAllTrainsV2 } from 'modules/simulationResult/components/SpaceTimeChart/d3Helpers';
-import { CHART_AXES } from 'modules/simulationResult/consts';
-import type { TimeScaleDomain } from 'modules/simulationResult/types';
-import type { Chart } from 'reducers/osrdsimulation/types';
-import { dateIsInRange, isoDateToMs, formatToIsoDate } from 'utils/date';
-
-import { SPACE_TIME_CHART_ID } from './consts';
-import useGetProjectedTrainOperationalPoints from './hooks';
-import type { DispatchUpdateSelectedTrainId } from './types';
-
-const CHART_MIN_HEIGHT = 250;
-
-/**
- * @summary A Important chart to study evolution of trains vs time and inside block occupancies
- *operationalPoints
- * @version 1.0
- *
- * Features:
- * - Possible to slide a train and update its departure time
- * - Type " + " or " - " to update departure time by second
- * - use ctrl + third mouse button to zoom it / out
- * - use shift + hold left mouse button to pan
- * - Right-Bottom bottom to switch Scales
- * - Resize vertically
- *
- */
-
-export type SpaceTimeChartV2Props = {
-  infraId?: number;
-  initialHeight?: number;
-  inputSelectedTrain: TrainScheduleResult;
-  trainIdUsedForProjection?: number;
-  simulation: TrainSpaceTimeData[];
-  simulationIsPlaying?: boolean;
-  timeScaleDomain?: TimeScaleDomain;
-  onSetBaseHeight?: (newHeight: number) => void;
-  dispatchUpdateSelectedTrainId: DispatchUpdateSelectedTrainId;
-  setTrainResultsToFetch?: (trainSchedulesIDs?: number[]) => void;
-  setTimeScaleDomain?: (newTimeScaleDomain: TimeScaleDomain) => void;
-  setTrainSpaceTimeData: React.Dispatch<React.SetStateAction<TrainSpaceTimeData[]>>;
-  deactivateChartSynchronization?: boolean;
-};
-
-const SpaceTimeChartV2 = ({
-  infraId,
-  initialHeight = 400,
-  inputSelectedTrain,
-  trainIdUsedForProjection,
-  simulation,
-  simulationIsPlaying = false,
-  timeScaleDomain,
-  onSetBaseHeight = noop,
-  dispatchUpdateSelectedTrainId,
-  setTrainResultsToFetch = noop,
-  setTimeScaleDomain,
-  setTrainSpaceTimeData,
-  deactivateChartSynchronization,
-}: SpaceTimeChartV2Props) => {
-  const ref = useRef<HTMLDivElement>(null);
-  const rndContainerRef = useRef<Rnd>(null);
-  const { t } = useTranslation('simulation');
-
-  const [baseHeight, setBaseHeight] = useState(initialHeight);
-  const [chart, setChart] = useState<Chart | undefined>();
-  const [dragOffset, setDragOffset] = useState(0);
-  const [height, setHeight] = useState(initialHeight);
-  const [resetChart, setResetChart] = useState(false);
-  const [rotate, setRotate] = useState(false);
-  const [selectedTrain, setSelectedTrain] = useState(inputSelectedTrain);
-  const [showModal, setShowModal] = useState<'+' | '-' | ''>('');
-  const [trainSimulations, setTrainSimulations] = useState<TrainSpaceTimeData[]>(simulation);
-
-  const operationalPoints = useGetProjectedTrainOperationalPoints(
```

Comment:
You can remove this file: useGetProjectedTrainOperationalPoints.ts

Reply:
done